<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="createProjectButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
    background-color: #1E1E2F;
    border-radius: 12px;
    color: white;
    padding: 10px;
    text-align: left;       /* Вирівнювання тексту по лівому краю */
    font-size: 12px;
}
QToolButton:hover {
    background-color: #3A3A5C;
    color: #FFD700;
}</string>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="importProjectButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
    background-color: #1E1E2F;
    border-radius: 12px;
    color: white;
    padding: 10px;
    text-align: left;       /* Вирівнювання тексту по лівому краю */
    font-size: 12px;
}
QToolButton:hover {
    background-color: #3A3A5C;
    color: #FFD700;
}</string>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="helpMainButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>250</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {
    background-color: #1E1E2F;
    border-radius: 12px;
    color: white;
    padding: 10px;
    text-align: left;       /* Вирівнювання тексту по лівому краю */
    font-size: 12px;
}
QToolButton:hover {
    background-color: #3A3A5C;
    color: #FFD700;
}</string>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Audio Editor</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
